plugins/uefi-recovery/uefi-recovery.quirk

```diff
@@ -0,0 +1,3 @@
+# Silicom Minnowboard Turbot MNW2MAX1.X64.0100.R01.1811141729
+[HwId=ea358e00-39f1-55b6-97be-a39225a585e1]
+Flags = requires-uefi-recovery
```

Comment:
How do you prevent this recovery device from getting re-created after upgrading to a new firmware with all the fixes? Do your need a better than clause?

Reply:
Good question. That HwId match wouldn't happen, as the 3rd CHID would be different, e.g.

    {ea358e00-39f1-55b6-97be-a39225a585e1}   <- Manufacturer + ProductName + BiosVendor + BiosVersion + BiosMajorRelease + BiosMinorRelease

where:

    BiosVersion: MNW2MAX1.X64.0100.R01.1811141729

Reply:
So they're going to need to have their capsule have two GUIDs that it matches then basically:

* The recovery HwID from the busted firmware
* The actual ESRT GUID

Reply:
Right. The way I envisioned this working was to have the HwID for version 1.0.1 only and then do a 1.0.1 -> {newest} with just the ESRT GUID.